app/MetricUnitConverter.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace ConverterApp;
+
+class MetricUnitConverter extends Converter
+{
+    public function converter($currentUnit, $newUnit)
+    {
+        $result;
+        $calculatedPower = $currentUnit->getUnit()->getPower() - $newUnit->getUnit()->getPower();
+        $finalPower = pow(10, $calculatedPower);
```

Comment:
This is not power but a multiplier or conversion coefficient. Its name should be adjusted accordingly.